src/main/java/org/takes/rq/RqForm.java

```diff
@@ -183,11 +215,13 @@ public Smart(final RqForm req) {
             this.origin = req;
         }
         @Override
-        public Iterable<String> param(final CharSequence name) {
+        public Iterable<String> param(
+            final CharSequence name
```

Comment:
@lautarobock This param can be in the line above (you can declare `throws` in this line instead).